docker-removeall.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+# Description:
+#   Prepare Docker for a fresh start, removing most everything:
+#     all composer projects, containers and anonymous volumes
+#     all non-composer containers
+#     all images, etc. per docker system prune ...
+
+MAX_WAIT=45  # Max time to wait for compose project to exit
+EACH_WAIT=5  # Sleep time between iterations of waiting for compose project to exit
+
+ENV=/usr/bin/env
+DOCKER="$ENV docker"
+TIMEOUT_MINIMAL=1
+
+# Currently "Docker Desktop" bundles compose as part of docker.
+# Earlier distributions, including those on servers, would likely require
+# that docker-compose be installed.  Find which we have... and if not found
+# then we will avoid any compose-based cleanup but still prune docker, etc.
+COMPOSE=""
+for i in 'docker compose' docker-compose ; do
+  ver=$( $i version 2> /dev/null )
+  if [ 0 -eq $? ] && [[ "$ver" =~ ^Docker\ Compose\ version\ v ]] ; then
+    COMPOSE="$i"
+    break
+  fi
+done
```

Comment:
I don't think this is working as expected.  I am troubleshooting on the box directly as I ran into

```
Could not identify docker-compose (or docker with compose)...
  skipping docker-compose cleanup bits!
```

But I do see it on the box

```
$ which docker-compose
/usr/bin/docker-compose
$ which docker
/usr/bin/docker
```

Reply:
Looks like the output for docker-compose version is `docker-compose version 1.27.4, build unknown`